autoscheduler/scraper/urls.py

```diff
@@ -1,6 +1,12 @@
 from django.urls import path
 from scraper.views import RetrieveCourseView
+from scraper.views import ListSectionView
+from scraper.views import RetrieveTermView
+from scraper.views import RetrieveCourseSearchView
 
 urlpatterns = [
     path('course', RetrieveCourseView.as_view()),
+    path('sections', ListSectionView.as_view()),
```

Comment:
I included them so that pytest would run because I was getting import errors. I'll remove them once I make the suggested changes and make sure everything still runs correctly.